test-projects/lambda-client/amplify/functions/todo-count/handler.ts

```diff
@@ -1,14 +1,16 @@
 import type { Handler } from "aws-lambda";
-import { Amplify } from "aws-amplify";
-import { generateClient } from "aws-amplify/data";
-import type { Schema } from "../../data/resource";
-import { resourceConfig, libraryOptions } from "$amplify/client-config/todo-count"
-
-Amplify.configure(resourceConfig, libraryOptions);
-
-const client = generateClient<Schema>();
+// import { Amplify } from "aws-amplify";
+// import { generateClient } from "aws-amplify/data";
+// import type { Schema } from "../../data/resource";
+// import { resourceConfig, libraryOptions } from "$amplify/client-config/todo-count"
+//
+// Amplify.configure(resourceConfig, libraryOptions);
+//
+// const client = generateClient<Schema>();
 
 export const handler: Handler = async () => {
-  const todos = (await client.models.Todo.list()).data;
-  return todos.length;
+  // const todos = (await client.models.Todo.list()).data;
+  // return todos.length;
+
+  console.log(process.env.MIS);
```

Comment:
it works as well.

![image](https://github.com/user-attachments/assets/92037447-c273-4437-9545-6a4b68575a1f)